### PR TITLE
[KB-278] Add paper about ANN in related works

### PR DESCRIPTION
**Pull request by Pattrigue**

### CODE REVIEW

File: report_thesis/src/sections/related_work.tex [hunk 1]
```diff
@@ -12,6 +12,10 @@ \subsection{Machine Learning Models in LIBS Analysis}
 They demonstrated that blending different models, such as \gls{gbr} and \gls{pls} for \ce{SiO2}, could improve prediction accuracy.
 Their study serves as a benchmark for model performance on LIBS spectra and offers insights into model selection for similar datasets.
 
+\citet{el_haddad_ann_2013} explored the application of \gls{ann} for quantitative analysis of soil samples using \gls{libs}, employing a three-layer perceptron \gls{ann} architecture to address matrix effects and nonlinearities.
+They demonstrated that \gls{ann} is efficient for predicting the concentrations of \ce{Al}, \ce{Ca}, \ce{Cu}, and \ce{Fe}.
+Incorporating spectral lines from other chemical elements in addition to the analyte of interest was also shown to significantly improve predictive accuracy.
```
Comment (chhoumann) [inline report_thesis/src/sections/related_work.tex:17]: What does this mean? Does it refer to predicting multiple oxides at once? What does it mean to incorporate spectral lines from other chemical elements? Do they remove some?

Comment (chhoumann) [reply report_thesis/src/sections/related_work.tex:17]: @Ivikhostrup told me that it's basically "more data, better model," which I think could play well with the challenge of limited data that we highlight. So saying that more explicitly would be good?

**Review by chhoumann — COMMENTED** [review]
See comment

File: report_thesis/src/sections/related_work.tex (hunk 1 above)
Comment (Pattrigue) [inline report_thesis/src/sections/related_work.tex:17]:
```suggestion
Incorporating additional spectral lines from other chemical elements, thereby increasing the amount of data input to the model, was also shown to significantly improve predictive accuracy.
```

**Review by chhoumann — APPROVED** [review]
Looks good

**Pull request merged**